SETUP.md

```diff
@@ -0,0 +1,54 @@
+# Setup
+
+## Requirements
+
+- Docker
+- Golang 1.19.2+
```

Comment:
we only use 1.18 in our go.mod so i think let's put 1.18?

Reply:
+1

Reply:
Addressed ✅